framework/src/ble_manager/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config BLE_MANAGER
+    bool "Enable BLE Easy Manager"
+    default n
+	select PIPES
+    depends on DRIVERS_BLE
+    ---help---
+        Easy APIs for app to use and control BLE features
```

Comment:
All APIs should be easy enough to use. So saying 'easy' APIs is not necessary.

Reply:
Ok I will fix it